lib/rides/models/ride.dart

```diff
@@ -65,22 +65,22 @@ class Ride extends Trip {
   @override
   factory Ride.fromJson(Map<String, dynamic> json) {
     return Ride(
-      id: json['id'],
-      createdAt: DateTime.parse(json['created_at']),
-      start: json['start'],
-      startPosition: Position(json['start_lat'].toDouble(), json['start_lng'].toDouble()),
-      startTime: DateTime.parse(json['start_time']),
-      end: json['end'],
-      endPosition: Position(json['end_lat'].toDouble(), json['end_lng'].toDouble()),
-      endTime: DateTime.parse(json['end_time']),
-      seats: json['seats'],
-      price: json['price'],
-      status: RideStatus.values[json['status']],
-      riderId: json['rider_id'],
-      rider: json.containsKey('rider') ? Profile.fromJson(json['rider']) : null,
-      driveId: json['drive_id'],
-      drive: json.containsKey('drive') ? Drive.fromJson(json['drive']) : null,
-    );
+        id: json['id'],
+        createdAt: DateTime.parse(json['created_at']),
+        start: json['start'],
+        startPosition: Position(json['start_lat'].toDouble(), json['start_lng'].toDouble()),
+        startTime: DateTime.parse(json['start_time']),
+        end: json['end'],
+        endPosition: Position(json['end_lat'].toDouble(), json['end_lng'].toDouble()),
+        endTime: DateTime.parse(json['end_time']),
+        seats: json['seats'],
+        price: json['price'],
+        status: RideStatus.values[json['status']],
+        riderId: json['rider_id'],
+        rider: json.containsKey('rider') ? Profile.fromJson(json['rider']) : null,
+        driveId: json['drive_id'],
+        drive: json.containsKey('drive') ? Drive.fromJson(json['drive']) : null,
+        hideinListView: json['hide_in_listview']);
```

Comment:
hide_in_listview -> hide_in_list_view

Reply:
I changed that so we can unblock the admin PR (btw quick plug, please review!!)